openresty-build-tools/kong-ngx-build

```diff
@@ -588,6 +594,7 @@ main() {
           "--with-http_stub_status_module"
           "--with-http_v2_module"
           "--without-http_encrypted_session_module"
+          "--with-luajit-xcflags='-DLUAJIT_VERSION=\\\"v${lj_version_with_time}\\\"'"
```

Comment:
Is the `v` needed here? Will it make `LUAJIT_VERSION` `vLuaJIT-2.1-20210510` instead?